test/releaseNotes.test.js

```diff
@@ -36,3 +42,54 @@ tap.test('Should return the correct PR numbers', async () => {
 
   tap.same(result, expected)
 })
+
+tap.test('Should return truncated PR body', async () => {
+  const tpl = fs.readFileSync(path.join(__dirname, '../src/pr.tpl'), 'utf8')
+
+  const testReleaseNotes = `
+    ## Whats Changed\n +
+    * chore 15 by @people in https://github.com/owner/repo/pull/13\n
+    * chore 18 by @people in https://github.com/owner/repo/pull/15\n
+    * chore 19 by @people in https://github.com/owner/repo/pull/16\n
+    * chore 21 by @people in https://github.com/owner/repo/pull/18\n
+    * fix 26 by @people in https://github.com/owner/repo/pull/42\n
+    * feature 30 by @people in https://github.com/owner/repo/pull/50\n
+    * fix 27 by @people in https://github.com/owner/repo/pull/52\n
+    * fix 32 by @people in https://github.com/owner/repo/pull/53\n
+    \n
+    \n
+    ## New Contributors\n
+    * @people made their first contribution in https://github.com/owner/repo/pull/13\n
+    * @people made their first contribution in https://github.com/owner/repo/pull/16\n
+    * @people made their first contribution in https://github.com/owner/repo/pull/42\n
+    * @people made their first contribution in https://github.com/owner/repo/pull/53\n
+    \n
+    \n
+    ## New documentation\n
+    * Link: https://somewhere.com/on/the/internet
+    \n
+    \n
+    **Full Changelog**: https://github.com/owner/repo/compare/v1.0.20...v1.1.0
+`
+
+  let longPrBody = testReleaseNotes
+  for (let i = 0; i < 70; i++) {
+    longPrBody = longPrBody + testReleaseNotes
+  }
+
+  tap.ok(longPrBody.length > 60000)
+
+  const truncatedPrBody = getPRBody(_template(tpl), {
+    newVersion: '1.0.0',
+    draftRelease: { id: 1, body: longPrBody },
+    inputs: [],
+    author: 'test',
+    artifact: null,
+  })
+  tap.ok(truncatedPrBody.length < 65536)
+  tap.ok(
```

Comment:
this is not the right way to assert here (it was probably a mistake in existing tests too). you should use the first argument provided to the test function instead